src/snowflake/snowpark/window.py

```diff
@@ -161,6 +255,41 @@ def _spec() -> "WindowSpec":
     rowsBetween = rows_between
 
 
+def _fill_window_spec_ast_with_relative_positions(
+    ast: proto.SpWindowSpecExpr,
+    start: Union[int, WindowRelativePosition],
+    end: Union[int, WindowRelativePosition],
+) -> None:
+    """
+    Helper function for AST generation to fill relative positions for window spec range-between, and rows-between. If value passed in for start/end is
+    of type WindowRelativePosition encoding will preserve for e.g. Window.CURRENT_ROW the syntax.
+
+    Args:
+        ast: AST where to fill start/end.
+        start: relative start position, integer or special enum value.
+        end: relative end position, integer or special enum value.
+    """
+    if isinstance(start, WindowRelativePosition):
+        if start == WindowRelativePosition.CURRENT_ROW:
```

Comment:
@sfc-gh-jdu can you review this part. I know you added additional window features last quarter.